ansible/roles/run-test-target/vars/main.yml

```diff
@@ -1,6 +1,6 @@
 ---
 
 integration_tests_root: "{{ collector_root }}/integration-tests"
-remote_tests_root: /tmp/collector-tests/
+remote_tests_root: "{{ ansible_env.HOME }}/collector-tests/"
```

Comment:
[ultra-nit] Think you can remove the trailing `/`, it's causing the tests to look for `collector-tests//collector-tests`